sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/Last.scala

```diff
@@ -42,6 +42,17 @@ case class Last(child: Expression, ignoreNullsExpr: Expression) extends Declarat
 
   override def children: Seq[Expression] = child :: Nil
 
+  // SPARK-16648: Default `TreeNode.withNewChildren` implementation doesn't work for `Last` when
```

Comment:
Well, it's probably unnecessary, since expressions are not part of the public API and won't appear in the generated ScalaDoc anyway...